packages/credential-provider-assume-role/src/fromTokenFile.ts

```diff
@@ -0,0 +1,126 @@
+import { AssumeRoleCommandInput, AssumeRoleWithWebIdentityCommandInput } from "@aws-sdk/client-sts";
+import { getMasterProfileName, parseKnownFiles, SourceProfileInit } from "@aws-sdk/credential-provider-ini";
+import { ProviderError } from "@aws-sdk/property-provider";
+import { ParsedIniData, Profile } from "@aws-sdk/shared-ini-file-loader";
+import { CredentialProvider, Credentials } from "@aws-sdk/types";
+import { readFileSync } from "fs";
+
+export const ENV_TOKEN_FILE = "AWS_WEB_IDENTITY_TOKEN_FILE";
+export const ENV_ROLE_ARN = "AWS_ROLE_ARN";
+export const ENV_ROLE_SESSION_NAME = "AWS_ROLE_SESSION_NAME";
```

Comment:
Do these really need to be exported? If not, let's make the API reference cleaner.

Reply:
They don't need to be exported. I'll remove them in next revision.

Reply:
The export for `ENV_*` constants was removed in [`59eecc8` (#2147)](https://github.com/aws/aws-sdk-js-v3/pull/2147/commits/59eecc85dc4058fc030c93126da83749265bd513)